Makefile

```diff
@@ -406,7 +406,7 @@ dev.shell.insights:
 	docker compose exec insights env TERM=$(TERM) bash -c 'eval $$(source /edx/app/insights/insights_env; echo PATH="$$PATH";) && /bin/bash'
 
 dev.shell.%: ## Run a shell on the specified service's container.
-	docker compose exec $* /bin/bash
+	docker compose exec -u 0 $* /bin/bash
```

Comment:
We need to be logging developers into the containers as root, NOT as the unprivileged app user.  Without root, you're basically helpless as you can't install custom requirements and other basic things like run management commands from the shell.